app/utils/initializer.js

```diff
@@ -1,10 +1,28 @@
 require('../config');
-require('../services/elasticSearch');
 
-const loggerInitialization = () => Promise.resolve();
+const logger = require('../utils/logger')('initializer');
+const { elasticSearchClient } = require('../services/elasticSearch');
+
+const elasticSearchInitialization = () => new Promise((resolve, reject) => {
+  elasticSearchClient.ping({
+    requestTimeout: 1000
+  }, (err) => {
+    if (err) {
+      logger.error(`elasticsearch connection error: ${err}`);
+      reject(err);
+    } else {
```

Comment:
in my opinion, i should say that we can refuse `else` statement, you can `return` one of callback functions. i'm talking about `resolve` and `reject`

Reply:
Yeap. I [update](https://github.com/inst-event-emitter/eventemitter-backend/pull/13/commits/03b617748894f02a8f8d8baa3f39a267c570be3d#diff-96a8661bff3c7b9cefe5405b208b2830R14) this code